### DISCUSSION TITLE
just for information

### DISCUSSION THREAD

**Discussion by rahulnitsxr2016** [incidentbot/incidentbot#155] [General]
Hi Team ,

Can you put a tagged version here which has only the slackbot functionality without any flask setup /web app functionality .

I want to test and use it for demo purpose without setting up any integrations at all . How can i do that? . 
probably i just want to test it in a sandbox without integration and without any UI. Just wanted to understand how to go about that.
basically i want it to work only in slack not on UI . :) .

**Comment by echoboomer** [comment]
@rahulnitsxr2016 is this still something you're interested in?

**Comment by rahulnitsxr2016** [comment]
Hi Echoboomer,

Thanks for your reply. Well Sure , if you can just guide me on the high
level steps to achieve this , it would be great help .
Appreciate your help

Regards
Rahul S

**Comment by echoboomer** [comment]
At this point, release `v2.0.0` is only backend, no frontend.
